metrics/lib/common.bash

```diff
@@ -281,6 +281,23 @@ function kill_processes_before_start() {
 	fi
 }
 
+# This function checks for active processes that were
+# left behind by cc-runtime.
+function check_no_processes_up() {
+	wait_time=5
+	while [ "$wait_time" -gt 0 ]; do
+		if [ ! "$(check_active_process ${HYPERVISOR_PATH})" ] && \
```

Comment:
I think that this check should be changed to something like:
https://github.com/clearcontainers/tests/blob/master/metrics/lib/common.bash#L267-L268